cfme/tests/cloud/test_azure_manual.py

```diff
@@ -0,0 +1,235 @@
+import pytest
+
+pytestmark = [pytest.mark.manual]
+
+
+@pytest.mark.manual
+def test_vpc_env_selection():
+    """
+    Test selection of components in environment page of cloud instances
+    with and without selected virtual private cloud
+    Related to BZ 1315945
+
+    Polarion:
+        assignee: anikifor
+        casecomponent: WebUI
+        initialEstimate: 1d
```

Comment:
are you sure `1d`?

Reply:
@digitronik I just moved the test, changing it to more realistic `1/2h`